templates/main.html

```diff
@@ -209,7 +209,8 @@ <h3>
     </div>
     <footer>
       <small data-i18n="gci-copyright">Google Code-in and the Google Code-in logo are trademarks of Google Inc.</small><br />
-      <small><span data-i18n="gitlab-copyright-1">The GitLab logo is licensed under</span> <a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a> <span data-i18n="gitlab-copyright-2">by GitLab Inc.</span></small>
+      <small><span data-i18n="gitlab-copyright-1">The GitLab logo is licensed under</span> <a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a> <span data-i18n="gitlab-copyright-2">by GitLab Inc.</span></small><br />
```

Comment:
`<br>` is enough.

See [here](https://stackoverflow.com/a/1946446/6914498).

Reply:
We have the backslash on the line above. Maybe keep it consistent for now?

Reply:
<br/>, <br /> will both work and  for consistency  I put <br /> as previous line used it. please don't make this undue  complicated. please review and close the task if it is good.

Reply:
You still have to merge your 3 commits @pshreya2181 do it then we'll need a maintainer soon to merge this PR 😉

Reply:
@andrewda While some `hr` elements not using backslash, there is some `img` elements using backslash. We need to fix this later. Consistency is important.